_glossary/core-config.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Core config
+type: markdown
+---
+Istio core component configs. Typically the configs to install Pilot, Proxy,
```

Comment:
Core config and Deployment config descriptions feel very similar. Can we add specific examples to each?